lib/validation/normalizeNode.js

```diff
@@ -10,31 +10,31 @@ type Normalizer = Change => any;
  * Create a schema definition with rules to normalize lists
  */
 function normalizeNode(opts: Options): Node => void | Normalizer {
-    return node => joinAdjacentLists(opts, node);
+    return (node, next) => joinAdjacentLists(opts, node, next);
```

Comment:
I think `next()` shouldn't leave the middleware functions.. So you can do something like
```js
return (node, next) => joinAdjacentLists(opts, node) || next()
```